### PR TITLE
Fix Az.Peering Syntax Errors

### PR DESCRIPTION
**Pull request by v-yuzhichen**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Peering/Peering/help/Remove-AzPeeringServicePrefix.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Remove a prefix from a peering service resource id.
 
 ### Example 3
 ```powershell
-Remove-AzPeeringServicePrefix -ResourceGroupName $peeringServiceGroup -PeeringServiceName $peeringServiceName -Name $prefixName -PassThru
+Remove-AzPeeringServicePrefix -ResourceGroupName $peeringServiceGroup -Name $prefixName -PassThru
```
Comment (ziyuezh576) [inline src/Peering/Peering/help/Remove-AzPeeringServicePrefix.md:58]: 40: Get-AzPeeringServicePrefix -ResourceGroupName $rgName -PeeringServiceName $peeringServiceName -Name $prefixName | Remove-AzPeeringServicePrefix

47: Remove-AzPeeringServicePrefix -ResourceId $peeringServicePrefixResourceId -PassThru

58: Remove-AzPeeringServicePrefix -ResourceGroupName $peeringServiceGroup -Name $peeringServiceName -PrefixName $prefixName -PassThru

File: src/Peering/Peering/help/Update-AzPeering.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Sets the Md5 Authentication Key
 
 ### Example 2: Update UseForPeeringService
 ```powershell
-Update-AzPeering -ResourceGroupName rg1 -Name ContosoPeering -UseForPeeringService $true
+Update-AzPeering -ResourceGroupName rg1 -Name ContosoPeering
```
Comment (ziyuezh576) [inline src/Peering/Peering/help/Update-AzPeering.md:67]: delete the entire example

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

File: src/Peering/Peering/help/Update-AzPeering.md (hunk 2 above)
Comment (v-yuzhichen) [reply src/Peering/Peering/help/Update-AzPeering.md:67]: Why delete the entire example?

Comment (ziyuezh576) [reply src/Peering/Peering/help/Update-AzPeering.md]: Without "UseForPeeringService" parameter, this example is meaningless. It updates nothing.

**Review by ziyuezh576 — APPROVED** [review]

**Pull request merged**